### ISSUE TITLE
Syntax Error, if ";" at last statement.

### ISSUE THREAD

**Issue by NilsHaldenwang** [bakkdoor/fancy#15]
```
ARGV for_option: "-r" do: |eval_string| {
  eval_string println
};
```

works, while

```
ARGV for_option: "-r" do: |eval_string| {
  eval_string println;
};
```

does not. 

Errormessage was: "ERROR [boot.fnc:21] : syntax error at symbol }", but the Error was in line 50.

**Comment by bakkdoor** [comment]
yeah, i'll make semicolons optional in these cases as well. reduces the amount of confusion. thanks!

**Comment by bakkdoor** [comment]
Changed parser to allow empty expressions (semicolons also at the end of a block etc.) since it caused unnecessary problems before. Closed by df7f82519e4f4cbb59e69dfcf90ef58cd6d3dc6d
